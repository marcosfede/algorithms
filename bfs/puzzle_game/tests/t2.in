3

1 9 2
8 3 7
4 6 5

4 5 6
1 2 3
7 8 9

7 8 9
1 2 3
4 5 6