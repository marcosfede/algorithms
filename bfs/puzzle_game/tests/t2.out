52
21
-1